src/common/HPM.c

```diff
@@ -207,6 +209,7 @@ bool hplugin_data_store_validate(enum HPluginDataTypes type, struct hplugin_data
 	struct hplugin_data_store *store;
 	nullpo_retr(false, storeptr);
 	store = *storeptr;
+	nullpo_retr(false, store);
```

Comment:
`store` can be NULL here (it's checked below). The function is a bit convoluted (my fault), but all cases should be handled.